src/components/forms/AcademicGrantsForm.tsx

```diff
@@ -0,0 +1,594 @@
+                  options={TIMEZONE_OPTIONS}
+                  onChange={option => {
+                    onChange((option as typeof TIMEZONE_OPTIONS[number]).value);
+                    trigger('timezone');
+                  }}
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
+                />
+              </Field>
+            )}
+          />
+
+          <TextField
+            id='projectName'
+            label='Project name'
+            helpText='This should be a concise description of the title of your project'
+            isRequired
+          />
+
+          <TextAreaField
+            id='projectDescription'
+            label='Brief project summary'
+            helpText="Describe your project in a few sentences (you'll have the chance to go into more
+              detail in the long form). If it's already underway, provide links to any existing
+              published work"
+            isRequired
+          />
+
+          <UploadFile
+            id='proposalAttachment'
+            label='Proposal'
+            title='Upload proposal'
+            helpText={
+              <>
+                Attach a PDF proposal. An proposal template is available{' '}
+                <Link
+                  fontWeight={700}
+                  color='brand.orange.100'
+                  href='https://notes.ethereum.org/@YlnZmB-mQ9ah8neq5cLDDg/Grant_Proposal_template'
+                  isExternal
+                  _hover={{ textDecoration: 'none' }}
+                >
+                  here
+                </Link>
+              </>
+            }
+            isRequired
+            onDrop={handleDrop}
+            mb={8}
+          />
+
+          <Controller
+            name='projectCategory'
+            control={control}
+            render={({ field: { onChange }, fieldState: { error } }) => (
+              <Field
+                id='projectCategory'
+                label='Project category'
+                helpText='Please choose a category that your project best fits in'
+                error={error}
+                isRequired
+              >
+                <Select
+                  id='projectCategory'
+                  options={ACADEMIC_GRANTS_PROJECT_CATEGORY_OPTIONS}
+                  onChange={option => {
+                    onChange(
+                      (option as typeof ACADEMIC_GRANTS_PROJECT_CATEGORY_OPTIONS[number]).value
+                    );
+                  }}
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
+                />
+              </Field>
+            )}
+          />
+
+          <TextField
```

Comment:
Field missing the error PageText when its not completed (eg: see Select fields on PSE form esp.ethereum.foundation/pse-sponsorships/apply)